lib/node_modules/@stdlib/assert/is-same-booleanarray/benchmark/benchmark.length.js

```diff
@@ -0,0 +1,110 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2024 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var bench = require( '@stdlib/bench' );
+var isBoolean = require( '@stdlib/assert/is-boolean' ).isPrimitive;
+var pow = require( '@stdlib/math/base/special/pow' );
+var Boolean = require( '@stdlib/boolean/ctor' );
+var BooleanArray = require( '@stdlib/array/bool' );
+var pkg = require( './../package.json' ).name;
+var isSameBooleanArray = require( './../lib' );
+
+
+// FUNCTIONS //
+
+/**
+* Creates a benchmark function.
+*
+* @private
+* @param {PositiveInteger} len - array length
+* @returns {Function} benchmark function
+*/
+function createBenchmark( len ) {
+	var x;
+	var y;
+	var v;
+	var i;
+
+	x = [];
+	y = [];
+	for ( i = 0; i < len; i++ ) {
+		v = Boolean( i%2 );
+		x.push( v );
+		y.push( v );
+	}
+	x = new BooleanArray( x );
+	y = new BooleanArray( y );
```

Comment:
```suggestion
	for ( i = 0; i < len; i++ ) {
		x.push( Boolean( i%2 ) );
	}
	x = new BooleanArray( x );
	y = new BooleanArray( x );
```

You only need one source array.